strawberry_django/fields/types.py

```diff
@@ -103,6 +103,91 @@ class ManyToManyInput:
         }
     )
 
+geo = False
+try:
+    from django.contrib.gis import geos
+    from django.contrib.gis.db import models as geos_fields
+
+    geo = True
+
+except django.core.exceptions.ImproperlyConfigured:
+    # If gdal is not available, skip.
+    pass
+
+if geo:
```

Comment:
I think this can be simplified by using an `else`. Also, it is better for typing because `geo` being `True` doesn't imply that the imports went fine from the typing point of view

```suggestion
try:
    from django.contrib.gis import geos
    from django.contrib.gis.db import models as geos_fields

except django.core.exceptions.ImproperlyConfigured:
    # If gdal is not available, skip.
    pass
else:
```